ktor-server/ktor-server-plugins/ktor-server-compression/jvm/src/io/ktor/server/plugins/compression/Compression.kt

```diff
@@ -91,6 +75,10 @@ public val Compression: RouteScopedPlugin<CompressionConfig> = createRouteScoped
 @OptIn(InternalAPI::class)
 private fun decode(call: PipelineCall, options: CompressionOptions) {
     val encodingRaw = call.request.headers[HttpHeaders.ContentEncoding]
+    if (call.isCompressionSuppressed) {
+        LOGGER.trace("Skip decompression for ${call.request.uri} because it is suppressed.")
```

Comment:
Shouldn't it be `call.isDecompressionSuppressed`

Reply:
fixed